ui/components-react/src/components-react/filter-builder/FilterBuilderLogicalOperator.tsx

```diff
@@ -0,0 +1,61 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from "react";
+import cx from "classnames";
+import { Anchor } from "@itwin/itwinui-react";
+import "./FilterBuilderLogicalOperator.scss";
+import { UiComponents } from "../UiComponents";
+
+type Operator = "And" | "Or";
+
+/**
+ * Props for [[PropertyFilterBuilderLogicalOperator]] component.
+ * @internal
+ */
+interface PropertyFilterBuilderLogicalOperatorProps {
+  /** Size to render the component. If undefined, defaults to iTwinUI "medium" size. */
+  size?: "small" | "large";
+  /** Allows toggling of operator by clicking operator text. */
+  isLinkDisabled?: boolean;
+  /** Operator to combine FilterBuilderRules. Must be either "And" or "Or". */
+  operator?: Operator;
+  /** Callback that is invoked when operator changes. */
+  onOperatorChange?: (operator: Operator) => void;
+}
+
+/** Component to render the operator inside of the filter builder
+ * @beta
+ */
+export const PropertyFilterBuilderLogicalOperator = (
+  props: Omit<React.HTMLProps<HTMLDivElement>, "size"> &
+    PropertyFilterBuilderLogicalOperatorProps
+) => {
+  const {
+    className,
+    size,
+    isLinkDisabled,
+    operator,
+    onOperatorChange,
+    ...rest
+  } = props;
+
+  const toggle = () => (operator === "And" ? "Or" : "And");
+
+  return (
+    <div
+      className={cx("fb-logical-operator", className)}
+      data-iui-size={size}
+      {...rest}
+    >
+      {undefined === operator ? (
+        <span>{UiComponents.translate("filterBuilder.group")}</span>
```

Comment:
What does `Group` means in context of this component? I don't think that whole `PropertyFilterBuilder` component should be kept generic (list of rules/criterias based on provided properties) and groups/grouping concept should be introduced in component that is built on top of `PropertyFilterBuilder`.

Reply:
so the idea is that a group is 2 or more rules such that the "+" button on each rule will add an additional rule to the group. The operator here is on the group level and would be "And" in the below screenshot.
![image](https://github.com/iTwin/appui/assets/85196633/9d25af74-7a61-4371-8944-72425e824927)
if we only want to support a singular rule in this component and have the higher order component handle the grouping, then we may need to rethink how rule addition will work since all of the addition exists within the rule/rule group components.

Reply:
Ah ok, sorry for the misunderstanding. Yes, the group string should be able to be removed.